ModTek/ModTek.cs

```diff
@@ -143,6 +143,21 @@ public static void Init()
             stopwatch.Stop();
         }
 
+        public static void Cleanup()
+        {
+            CloseLogStream();
+
+            modLoadOrder = null;
+            jsonMergeCache = null;
+            typeCache = null;
+            dbCache = null;
+
+            cachedJObjects = null;
```

Comment:
just thinking about this one line, this must be hundreds of MB for RT, have to check MechEngineer for such things